README.md

```diff
@@ -45,38 +45,38 @@ import 'package:system_theme/system_theme.dart';
 
 ### Get system accent color
 
-Use the getter `SystemTheme.accentInstance.accent` to get the system accent color.
+Use the getter `SystemTheme.accentColor.accent` to get the system accent color.
 
 ```dart
-final accentColor = SystemTheme.accentInstance.accent;
+final accentColor = SystemTheme.accentColor.accent;
 ```
 
 To reload the accent colors, use the method `load()`:
 
 ```dart
-await SystemTheme.accentInstance.load();
+await SystemTheme.accentColor.load();
 ```
 
 You can load the colors before running the app, so the colors can't be wrong at runtime:
 
 ```dart
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
```

Comment:
```suggestion
```

The `SystemAccentColor` already invoke the method `ensureInitialized`, meaning it needless to execute it beforehand.

Reply:
I'm not sure why `SystemAccentColor` invoke that method since it should be invoked at user-developer's code. For example, [firebase/flutterfire never call that in their lib code, but in their example code.](https://github.com/search?q=repo%3Afirebase%2Fflutterfire%20WidgetsFlutterBinding.ensureInitialized()&type=code)

So here may be another issue. This suggestion will not be accepted.